packages/react/src/ReactDebugCurrentFrame.js

```diff
@@ -7,26 +7,20 @@
  * @flow
  */
 
-import type {ReactElement} from 'shared/ReactElementType';
-
-import {describeUnknownElementTypeFrameInDEV} from 'shared/ReactComponentStackFrame';
-
 const ReactDebugCurrentFrame = {};
 
-let currentlyValidatingElement = (null: null | ReactElement);
+let currentExtraStackFrame = (null: null | string);
 
-export function setCurrentlyValidatingElement(element: null | ReactElement) {
+export function setExtraStackFrame(stack: null | string) {
   if (__DEV__) {
-    currentlyValidatingElement = element;
+    currentExtraStackFrame = stack;
   }
 }
 
 if (__DEV__) {
-  ReactDebugCurrentFrame.setCurrentlyValidatingElement = function(
-    element: null | ReactElement,
-  ) {
+  ReactDebugCurrentFrame.setExtraStackFrame = function(stack: null | string) {
```

Comment:
Since it's DEV it probably doesn't matter but in general that might not be great because one is meant to be called in this same package so it can be inlined. So that would avoid inlining it.